Integrations/QRadar/QRadar.py

```diff
@@ -1038,6 +1048,116 @@ def get_domains_by_id_command():
+    """
+        The function finds indicators according to user query and updates QRadar reference set
+    """
+    try:
+        args = demisto.args()
+        reference_name = args.get('ref_name')
+        element_type = args.get('element_type')
+        if not check_ref_set_exist(reference_name):
+            if element_type:
+                create_reference_set(reference_name, args.get('element_type'), args.get('timeout_type'),
+                                     args.get('time_to_live'))
+            else:
+                return_error("There isn't a reference set with the name {0}. To create a reference set,"
+                             " you have to enter an element type".format(reference_name))
+        else:
+            if element_type:
+                return_error("The reference set {0} is already exist. You are not supposed to enter element type,"
+                             " Try again".format(reference_name))
+        query = args.get('query')
+        value_indicators_list, indicators_data_list = get_indicators_list(query)
+        if len(value_indicators_list) == 0:
+            return {
+                'Type': entryTypes['note'],
+                'Contents': {},
+                'ContentsFormat': formats['json'],
+                'ReadableContentsFormat': formats['markdown'],
+                'HumanReadable': "No indicators found, Reference set {0} didn't change".format(reference_name)
+            }
+        else:
+            raw_response = upload_indicators_list_request(reference_name, value_indicators_list)
+            ref_set_data = unicode_to_str_recur(get_reference_by_name(reference_name))
+            indicator_headers = ['value', 'indicator_type']
+            ref_set_headers = ['name', 'element_type', 'timeout_type', 'creation_time', 'number_of_elements']
+            return {
+                'Type': entryTypes['note'],
+                'Contents': raw_response,
+                'ContentsFormat': formats['json'],
+                'ReadableContentsFormat': formats['markdown'],
+                'HumanReadable':
+                    tableToMarkdown("reference set {0} was updated".format(reference_name), ref_set_data,
+                                    headers=ref_set_headers) + tableToMarkdown("Indicators list",
+                                                                               indicators_data_list,
+                                                                               headers=indicator_headers)}
+    except Exception as e:
+        if '1005' in str(e):
+            return "You tried to add indicators that dont match to reference set type"
+        raise e
+
+
+def check_ref_set_exist(ref_set_name):
+    """
+        The function checks if reference set is exist
+
+    Args:
+        ref_set_name (str): Reference set name
+
+    Returns:
+        Dict. If found - Reference set object, else - Error
+    """
+
+    try:
+        return get_reference_by_name(ref_set_name)
```

Comment:
```suggestion
        return True if get_reference_by_name(ref_set_name) else False
```
Or change the name of the function

Reply:
so update the function name to get_ref_set

Reply:
done